jac-uk/apply
# Hotfix/1.73.3

## What's included?
This requirement came to use as a request from Paul O'Brien.

Added welsh translation to router for VacancyDetails.
Removed manual button to translate into welsh in VacancyDetails page.
Added welsh translations for text and title (for JAC00192 on prod).
Added welsh location text and hardcoded it in the Vacancies page.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

Carry out the test on the following vacancy on production: President and Deputy President of Adjudication Panel for Wales

1) Ensure the location has an equivalent welsh translation in the section below (see screenshot):

<img width="902" alt="Screenshot 2023-10-03 at 12 37 25" src="https://github.com/jac-uk/apply/assets/115651787/6d49c3bf-f52c-40a1-a442-41b6e78b22e8">

2) Ensure the following pieces of text are translated correctly into welsh on the vacancy details page:

- [The page title]
- Overview of the role
- Number of vacancies
- Salary/Fee
- Contact
- Relevant information (downloads)
- Timeline
- Open for applications
- Closed for applications
- About the role
- 28 September 2023
- 12 October 2023

3) Ensure the page title is correctly translated on the Task List page

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
